stable/atlantis/README.md

```diff
@@ -35,6 +35,7 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 | `environment`                               | Map of environment variables for the container.                                                                                                                                                                                                                                                           | `{}`    |
 | `imagePullSecrets`                          | List of secrets for pulling images from private registries.                                                                                                                                                                                                                                               | `[]`    |
 | `gitconfig`                                 | Contents of a file to be mounted to `~/.gitconfig`.  Use to allow redirection for Terraform modules in private git repositories.                                                                                                                                                                          | n/a     |
+| `command`                                   | Optionally override the arguments passed to the Atlantis Docker image. By default no arguments are passed which is the same as passing `["server"]`. Must be an array.                                                                                                                                  | `[]`    |
```

Comment:
Hmm, actually this isn't correct. According to the docs, `command` overrides `ENTRYPOINT` and `args` overrides  `CMD`

Reply:
(I see your comment now)